docs/api/03_plans/create-plan.mdx

```diff
@@ -429,7 +429,7 @@ curl --location --request POST "$LAGO_URL/api/v1/plans" \
     createdAt: null,
     lagoBillableMetricId: "id",
     chargeModel: "volume" as ChargeObject["chargeModel"],
-    instant: false,
+    pay_in_advance: false,
```

Comment:
```suggestion
    payInAdvance: false,
```

Reply:
Thanks for catching 😅 Fixed in all places